datastore/modelassertmanager.go

```diff
@@ -92,6 +127,134 @@ type ModelAssertion struct {
 	Modified      time.Time `json:"modified"`
 }
 
+// CreateModelAssertionHeaders returns the model assertion headers for a model
+func CreateModelAssertionHeaders(m Model) (map[string]interface{}, Keypair, error) {
```

Comment:
I understand that this wasmoved with the name from somewhere else, but in this package everything Create* operates mostly on the db. I wonder if this should just be called ModelAssertionHeadersForModel

Reply:
Yes, I moved this function from another package to avoid cyclic dependencies. It makes sense to rename this one